server/src/main/java/io/crate/fdw/JdbcBatchIterator.java

```diff
@@ -116,17 +122,20 @@ static String generateStatement(RelationName table,
             .append(table.name())
             .append(qs);
 
-        return String.format(
+        var stmt = String.format(
             Locale.ENGLISH,
             "SELECT %s FROM %s WHERE %s",
             String.join(", ", Lists.mapLazy(
                 columns,
                 ref -> new QuotedReference(ref, qs).toString(Style.UNQUALIFIED))),
             relationName.toString(),
             RefReplacer.replaceRefs(
-                query,
+                // Alias symbols must be expanded to their underlying symbols
+                AliasSymbolReplacer.replaceAliases(query, AliasSymbol::symbol),
```

Comment:
I thought normalize would already remove aliases (https://github.com/crate/crate/commit/fc1af565895c9203ccc1650061c6c6fefb07961d)

But maybe this is a regression due to https://github.com/crate/crate/commit/6e8eddac2cd0e25434c0143e4803920f9dbadb64 ? In any case - might be worth using the normalizer here instead? Or in the `ForeignCollect` operator. But this way is also fine.

Reply:
Good point. 
The problem here seems to be related to operator push downs. In this case the `MoveFilterBeneathRename` will convert a `ScopedSymbol` into a `AliasSymbol`. This happens after any normalization have taken place.
Shall we run the normalizer here (or in `ForeignCollect`) instead?

Reply:
Done by https://github.com/crate/crate/pull/16968/commits/9b9e3a4a76419d31ee578a16ce78c83d3864ee0b, pls check.